View_Assist_custom_sentences/Play_Music_with_Music_Assistant/blueprint-playmusicwithmusicassistant.yaml

```diff
@@ -24,152 +24,188 @@ blueprint:
+description: "Play music using music assistant"
 trigger:
   - platform: conversation
     command:
       - play the artist {artist}
-      - play some {artist}
-    id: playartist
+      - play some {artist} [music]
+      - play [some] [music | songs] by {artist}
+    id: artist
   - platform: conversation
     command:
       - start [the] {playlist} playlist
     id: playlist
   - platform: conversation
     command:
       - play {song} by {artist}
-    id: playsong
+    id: track
   - platform: conversation
     command:
       - queue {song} by {artist}
       - cue {song} by {artist}
-    id: cuesong
-condition: []
-action:
+    id: cuetrack
+conditions: []
+actions:
   - choose:
       - conditions:
           - condition: trigger
             id:
-              - playartist
+              - artist
         sequence:
-          - service: python_script.set_state
-            data:
-              entity_id: "{{ target_satellite_device }}"
-              mode: music
-              title: ""
-          - service: browser_mod.navigate
-            metadata: {}
-            data:
-              path: "{{ view_path }}"
-            target:
-              device_id: "{{ target_display_device }}"        
-          - service: music_assistant.search
-            data:
+          - data:
               limit: 1
               config_entry_id: "{{ ma_instance }}"
               media_type:
                 - artist
               name: "{{ trigger.slots.artist  }}"
             response_variable: found_artist
-          - service: music_assistant.play_media
-            target:
-              entity_id: "{{ target_musicplayer_device }}"
-            data:
-              media_type: artist
-              enqueue: replace
-              media_id: "{{ found_artist.artists.0.name }}"
-          - set_conversation_response: ""
+            action: music_assistant.search
+          - alias: Did we find an artist?
+            if:
+              - condition: template
+                value_template: "{{found_artist['artists'] | count>0}}"
+                alias: We found an artist!
+            then:
+              - data:
+                  entity_id: "{{ target_satellite_device }}"
+                  mode: music
+                  title: ""
+                action: python_script.set_state
+              - metadata: {}
+                data:
+                  path: "{{ view_path }}"
+                target:
+                  device_id: "{{ target_display_device }}"
+                action: browser_mod.navigate
+              - target:
+                  entity_id: "{{ target_musicplayer_device }}"
+                data:
+                  media_type: artist
+                  enqueue: replace
+                  media_id: "{{ found_artist.artists.0.name }}"
+                action: music_assistant.play_media
+                enabled: true
+              - set_conversation_response: "\"Playing {{ found_artist.artists.0.name }} music now!\""
```

Comment:
This needs to remain blank for now.  Folks using BrowserMod for their musicplayer_device will have issues.  BM does not handle playing annoucements over playing media files.   I plan to change this when I change the wiki to recommend HassMic for playback but for now this will need to be left as it was before to not break playback for others

Reply:
All other responses will need to be changed back to empty.  Perhaps you can comment the current and add the empty one so that when the switch is made it will be easy to swap to the verbose one.

I am getting the word 'quote' at the end of the spoken text.  I think you are adding one too many quote characters